doc/tech/roadmap.md

```diff
@@ -66,5 +66,45 @@ Please refer to individual [release plan](./release_notes/) and features for det
 <a name="3"></a>
 ## Dependencies between various Workstreams
 
+The various work streams in CNTT are 
+- Reference Model (RM)
+- Reference Architecture (RA)
+- Reference Implementation (RI) 
+- Reference Conformance (RC)
+
+The work stream dependency relationship in a simple terms, [Reference Conformance](../ref_cert) verifies and tests the [Reference Implementation](../ref_impl) which follows the requirements and architecture defined in the CNTT [Reference Architecture](../ref_arch) and Reference Architecture describes the high level system components and its interactions by adhering to the requirements and expectations set by the CNTT [Reference Model](../ref_model) which sets the standards for infrastructure abstraction, compliance and verification. 
+
+For the standard release stabilisation, the pattern in each release follows RM must be the advanced version of RA. Similarly RA must be the advanced version of RI and simultaneously for RC.
+
+There are two different track in CNTT
```

Comment:
I would suggest that we state the two tracks as supporting virtualised workloads and containerised workloads. The technology choices result from that.

Reply:
@pgoyal01 - I didn't get you. But I tried to follow the same terminology used in CNTT. Are you saying replace with "virtualised workloads" instead of "OpenStack based" and "containerised workload" instead of "Kubernetes based"?

Reply:
@prabhubalan This is a suggestion as it better reflects what it is about.

Reply:
Thanks, @kedmison. I agree.